OneCommunityGlobal/HighestGoodNetworkApp
# Zijie reducer lint fixed

Find bugs, do not review for now, will fix in the next week


## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as admin user

(Please be sure you record a video)
5. **Verify that the reducer is working normally.**
